www/app/assets/js/app.js

```diff
@@ -2195,20 +2244,20 @@ angular.module('kubernetesApp.components.dashboard')
     var pods = {
       "kind": "PodList",
       "creationTimestamp": null,
-      "selfLink": "/api/v1beta1/pods",
+      "selfLink": "/api/v1beta3/pods",
```

Comment:
This is all still v1beta1 -- desiredState, manifest, ... The v1beta3 schema is different. The example needs to be regenerated.